libs/state/src/lib/rx-state.service.ts

```diff
@@ -34,7 +34,7 @@ type ProjectValueReducer<T, K extends keyof T, V> = (
  * @description
  * RxState is a light-weight reactive state management service for managing local state in angular.
  *
- * ![state logo](https://raw.githubusercontent.com/BioPhoton/rx-angular/master/libs/state/images/state_logo.png)
+ * ![state logo](https://raw.githubusercontent.com/BioPhoton/rx-angular/master/docs/images/state/state_logo.png)
```

Comment:
I saw the issue about renaming master to main. Should we consider this here? Or would this introduce too much over these and we create a new issue for the steps todo?

Reply:
It might create some issues.

- All open PRs and GitHub pages targeting master. Not sure the target branch will be changed automatically.
- How it will affect git actions and other automation tools that also target master?
- We will need to update links (but that's not the biggest effort)

GitHub will release tools for seamless migration later this year. 
More info here: https://github.com/github/renaming